x-pack/plugins/apm/server/saved_objects/apm_custom_dashboards.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SavedObjectsType } from '@kbn/core/server';
+import { i18n } from '@kbn/i18n';
+import { schema } from '@kbn/config-schema';
+import { APM_CUSTOM_DASHBOARDS_SAVED_OBJECT_TYPE } from '../../common/service_dashboards';
+
+export const apmCustomDashboards: SavedObjectsType = {
+  name: APM_CUSTOM_DASHBOARDS_SAVED_OBJECT_TYPE,
+  hidden: false,
+  namespaceType: 'multiple',
+  mappings: {
+    properties: {
+      dashboardSavedObjectId: { type: 'keyword' },
+      kuery: { type: 'text' },
+      useContextFilter: { type: 'boolean' },
+    },
+  },
+  management: {
+    importableAndExportable: false,
```

Comment:
this should be true, people want to be able to export/import

Reply:
fixed here https://github.com/elastic/kibana/pull/166789/commits/d3c66fc5e32b066d973e78d9c4d69fdb14acd2f1